pyproject.toml

```
@@ -0,0 +1,10 @@
+[build-system]
+requires = ["setuptools", "tomli", "wheel"]
```

```suggestion
requires = ["setuptools", "versioneer[toml]"]
```

If you insist on using `versioneer`, make this change and delete `versioneer.py` from the root of the project.